app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
[REQUIRED] Kindly reconcile your login page with the project's wireframe. Also, ensure your signup page is the same as the wireframe.

Your current login page  👇

![student signin devise](https://github.com/hydaline-code/myblogapp/assets/66097371/550212a1-23a2-47d9-bd07-8b463bc388ab)

Given Project's wireframe 👇

![signin wireframe](https://github.com/hydaline-code/myblogapp/assets/66097371/a7ac1279-6d82-4d46-afa7-f6eb0f91167d)